.travis.yml

```diff
@@ -23,10 +25,16 @@ install:
     - popd && popd
     - git clone https://github.com/01org/TPM2.0-TSS.git
     - pushd TPM2.0-TSS
-    - git checkout 1.0
+    - git checkout 1.1.0_beta0
     - ./bootstrap && ./configure && make -j$(nproc) && sudo make install && popd
     - sudo ldconfig /usr/local/lib
-    - resourcemgr -sim -tpmhost 127.0.0.1 &
+    - git clone https://github.com/01org/tpm2-abrmd.git
+    - pushd tpm2-abrmd
+    - ./bootstrap && ./configure --with-dbuspolicydir=/etc/dbus-1/system.d && make -j$(nproc) && sudo make install && popd
+    - sudo mkdir -p /var/lib/tpm
+    - sudo groupadd tss && sudo useradd -M -d /var/lib/tpm -s /bin/false -g tss tss
+    - sudo pkill -HUP dbus-daemon
+    - sudo -u tss tpm2-abrmd --tcti=socket &
```

Comment:
In all fairness this is my doing. The dbus policy packaged with tpm2-abrmd will allow a process run as either root or the tss user to claim the Tabrmd name on the system bus. Setting up a tss user seemed like "the right thing to do" here. The only thing we can blame on dbus is having to send `SIGHUP` to the `dbus-daemon` so that it picks up the configuration installed by the `tpm2-abrmd` build.

I could get this down to two lines if the `tpm2-abrmd` is run as root.